bin/start-uc-server-in-docker

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -ex;
+
+container_name="unitycatalog"
+network_name="unitycatalog_network"
+network_details=$(docker network ls --format '{{json .}}' | jq -r --arg NETWORK_NAME "$network_name" 'select(.Name == $NETWORK_NAME) | .Name')
+container_details=$(docker container ls --format '{{json .}}' | jq -r --arg CONTAINER_NAME "$container_name" 'select(.Names == $CONTAINER_NAME) | .Names')
+container_runing=$(docker ps --format '{{json .}}' | jq -r --arg CONTAINER_NAME "$container_name" 'select(.Names == $CONTAINER_NAME) | .Names')
+
+# Create the network if it doesn't exist
+if [ -z "$network_details" ]; then
+    echo "Network $network_name does not exist. Creating it..."
+    docker network create $network_name
+    echo "Network $network_name created."
+else
+    echo "Network $network_name already exists, skipping."
+fi
```

Comment:
Network is not absolutely necessary but for compatibility across various operating systems, using a network is the most straight-forward way to interact between containers.

I have included some explanation in the documentation here's a snippet:

> [!IMPORTANT]
> Because the default run script runs the script on a network called `unitycatalog_network`, 
> if you want to test the catalog you need to make sure you application is running in the same network.
> For instance, you will not be able to test the API from your machine directly using Postman,
> wget or cUrl unless you run them with the catalog on the same network, or you change the catalog's
> network to run on the Host network.
> This is OS specific, therefore we opted for the most generic option, which is to create 
> a docker network, but feel free to change those configurations to suit your needs.